### ISSUE TITLE
Changes to not topics not overwritten

### ISSUE THREAD

**Issue by glyph-se** [github/safe-settings#343]
## Problem Description
I have configured safe-settings to apply a list of topics to be applied by safe-settings. This work good initially, but when I change the topics manually in the GitHub UI the topics are not applied again by safe-settings.

### What is actually happening
Nothing

### What is the expected behavior
I expect safe-settings to overwrite the list of topics within a few seconds.

### Error output, if available
I didn't see any error message when looking in the console, though it is possible that I missed some since our organization have a lot of activity.

### Are you using the hosted instance of probot/settings or running your own?
My own instance

#### Version of probot/settings
Using 028c368fd39eef6f97c24d43115a10eaf41cf2db

**Comment by glyph-se** [comment]
The same applies to labels. I tried adding the label webhook to the app, but no effect

**Comment by martinm82** [comment]
I think safe-settings does not listen to all events. If you for example change branch protection rules those will be almost immediately overridden.

You have to configure the cron job to update things like topics, labels, etc.

**Comment by glyph-se** [comment]
> I think safe-settings does not listen to all events. If you for example change branch protection rules those will be almost immediately overridden.
> 
> You have to configure the cron job to update things like topics, labels, etc.

Looks like we might need this longterm, though I would expect most things to work without that.

We are hosting it in an Azure function and it appears to be some work to setup cron there. I found another issue about doing this for Aws lambdas which required modifications to the code https://github.com/github/safe-settings/issues/313

**Comment by decyjphr** [comment]
> I think safe-settings does not listen to all events. If you for example change branch protection rules those will be almost immediately overridden.

Correct, it does not handle things certain changes like Topic changes because there are no webhook events for it. And for other changes, like Labels, there is a webhook event but it is not currently supported and we could add a handler for that.
